treadmill-scripts/configureldap.sh

```diff
@@ -116,9 +116,11 @@ trap cleanup EXIT
 cat << E%O%F > ${tmpdir}/LDAP.yaml
 ---
 realm: ${REALM}
+treadmill_cell: None
```

Comment:
None is literal string here. you may want to use -, which is translated to python None.

Reply:
Updated to use -